tests/test_summary.py

```diff
@@ -180,6 +180,58 @@ def test_res2csv_summary(tmp_path, mocker):
     assert str(disk_df["DATE"].values[-1]) == "2003-01-02"
 
 
+@pytest.mark.integration
+def test_res2csv_summary_no_output_on_missing_and_invalid_input(tmp_path, mocker):
+    """Test that no output file is created when the input file is missing or invalid."""
+
+    tmp_input_file = tmp_path / "MISSING_INPUT.UNSMRY"
+    tmp_output_file = tmp_path / "missing_output.arrow"
+
+    tmp_input_file.touch()
```

Comment:
(this can be confusing later, the filename indicates it is missing, but then it is not after you have touched it)